source/docs/bv_expriments/bitmap.rs

```diff
@@ -0,0 +1,178 @@
+#[allow(unused_imports)]
```

Comment:
Is this something that would verify with the current implementation? If so, consider moving it to examples/ so it is run by CI too.

Reply:
(Also, minor bv_expriments -> bv_experiments)

Reply:
This example currently hangs on the default setting. It works fine with `(set-option :smt.case_split 0)`. (Default value is 3)
I will make sure to move this to examples when I work on multiple backend supports.